src/api/spec/views/webui/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```

Comment:
No I don't think so. I mean what are you testing with this test? That `unless` doesn't break?